task-2/src/core/api/index.js

```diff
@@ -0,0 +1,3 @@
+export const moviesApiLink = () => {
+	return `http://localhost:4000/movies?limit=12`;
```

Comment:
This function is not needed anymore.
Also, go through the project and remove commented code that is not used.

Reply:
resolved.